patroni/__init__.py

```diff
@@ -164,7 +167,28 @@ def shutdown(self):
 
 
 def patroni_main():
-    patroni = Patroni()
+    config = __import__('patroni.config', fromlist=[""])
```

Comment:
Why not just `from .config import Config`?

Reply:
fixed, at some point it didn't work, maybe when there was a circular import